docs/real-user-metrics.md

```diff
@@ -0,0 +1,60 @@
+# Real User Metrics
+
+Real user metrics allow a user's browser to report on how a page loaded using the [Performance API]. The tool that we use to do this is called [LUX] - short for Live User Experience - and is run by Speedcurve.
+
+The benefit of RUM is that it shows how pages are performing in real situations, rather than on synthetic tests. This means that we don't have to guess at what conditions might be - RUM provides insight into what conditions actually are.
+
+## Loading
+
+The LUX scripts should only be loaded when a user has opted into usage tracking. This is done using the `rum-loader` script, which checks if the `usage` cookie is `true` and then loads the required scripts. This is already part of the [Public Layout component].
+
+## Custom changes when updating LUX
+
+The scripts for the real user metrics are loaded from our servers - this allows us to know what is contained within the scripts and reduce the risk of unwelcome things being added to GOV.UK
+
+Because of this, the scripts are audited before being updated and then two extra lines are added to make sure it works correctly. These two lines set the customer ID and turn on beacon mode.
```

Comment:
Can we say anything more specific about this audit and what is involved?

Reply:
I'll ask - and update the docs when I hear back. This shouldn't be a blocker for this pull request.

Reply:
Fair enough. Just so long as we don't forget.